packages/agents/src/tools/textToSpeech.ts

```diff
@@ -0,0 +1,30 @@
+import type { Tool } from "../types";
+
+export const textToSpeechTool: Tool = {
+	name: "textToSpeech",
+	description: "This tool takes a text input and turns it into an audio file.",
+	examples: [
+		{
+			prompt: 'Say the following out loud:"Hello world!"',
+			code: "textToSpeech('Hello world!')",
+			tools: ["textToSpeech"],
+		},
+		{
+			prompt: "Say the content of the string txt out loud",
+			code: "textToSpeech(txt)",
+			tools: ["textToSpeech"],
+		},
+	],
+	call: async (input, inference) => {
+		const data = await input;
+		if (typeof data !== "string") throw "Input must be a string.";
+
+		return inference.textToSpeech(
+			{
+				inputs: data,
+				model: "espnet/kan-bayashi_ljspeech_vits",
```

Comment:
is this the same model used in transformers Agents implem?

Reply:
Is is the [recommended model](https://github.com/huggingface/huggingface.js/blob/208e91408e058e80bd155d2f3143addd442c51bc/packages/inference/src/tasks/audio/textToSpeech.ts#L14-L17)  by `@huggingface/inference`

Reply:
yes that one is probably way better actually, `espnet/kan-bayashi_ljspeech_vits` is a bit old